cmd/operator-sdk/main.go

```diff
@@ -48,7 +48,7 @@ func main() {
 				"See `operator-sdk init -h` and the following doc on how to scaffold a new project:\n" +
 				"https://v0-19-x.sdk.operatorframework.io/docs/golang/quickstart/\n" +
 				"To migrate existing projects to the new layout see:\n" +
-				"https://sdk.operatorframework.io/docs/golang/migration/project_migration_guide/\n"
+				"https://v0-19-x.sdk.operatorframework.io/docs/golang/project_migration_guide/\n"
 			projutil.PrintDeprecationWarning(depMsg)
 		}
 		if err := cli.RunLegacy(); err != nil {
```

Comment:
It shows great and it is just missing the fragment. 
See the changelog directory. we add the fragment and teh CHANGELOG it is gen over it.

Reply:
Thanks @camilamacedo86 . I added the changelog fragment. Please review.